official/utils/flags/_benchmark.py

```diff
@@ -66,6 +66,12 @@ def define_benchmark(benchmark_log_dir=True, bigquery_uploader=True):
         help=help_wrap("The Bigquery table name where the benchmark run "
                        "information will be uploaded."))
 
+    flags.DEFINE_string(
+        name="bigquery_run_status_table", short_name="brst",
+        default="benchmark_run_status",
+        help=help_wrap("The Bigquery table name where the benchmark run "
+                       "status information will be uploaded."))
```

Comment:
Rather than requiring this as an arg, can we expect the name to be some derivative of the main table name? Or take a prefix and use it for all the tables? Ends up being a lot of args in at the command line otherwise.